openshift/scripts/common/envars

```diff
@@ -8,3 +8,5 @@ PROJ_PROD="${PROJ_PROD:-auzhsi-prod}"
 TAG_PROD="${TAG_PROD:-prod}"
 PATH_BC="${PATH_BC:-$(dirname ${0})/../templates/build.bc.yaml}"
 PATH_DC="${PATH_DC:-$(dirname ${0})/../templates/deploy.dc.yaml}"
+KEYCLOAK_PROD_URL="https://sso.pathfinder.gov.bc.ca/auth"
+KEYCLOAK_DEV_URL="https://sso-dev.pathfinder.gov.bc.ca/auth"
```

Comment:
I would name them SSO_BASE_PROD_URL and SSO_BASE_DEV_URL and btw I think you can just set the dev url here and use `https://sso.pathfinder.gov.bc.ca/auth` as a default in the template

Reply:
@Kyubinhan I want to stick with KEYCLOAK naming convention rather than SSO. Also I dont want app to point to prod by default.